dropbox/dropbox.py

```diff
@@ -372,6 +373,7 @@ def request_json_string(self,
                                data=body,
                                stream=stream,
                                verify=True,
+                               timeout=socket.getdefaulttimeout()
```

Comment:
Do you think it makes sense for the Dropbox client object to take an optional `timeout` argument that overrides the global one? You could use the default timeout if none is provided. IMO it's more likely for the socket to be set based on the connection, rather than globally for all connections.